bot/resources/evergreen/trivia_quiz.json

```
@@ -0,0 +1,221 @@
+    {
+      "id": 4,
+      "hints": ["It was revealed in the Nintendo Character Guide in 1993.", "His last name has to do with eating Mario's enemies."],
+      "question": "What is Yoshi's (from Mario Bros.) full name?",
+      "answer": "Yoshisaur Munchakoopas"
+    },
+    {
+      "id": 5,
+      "hints": ["The game was released in 1990.", "It was released on the SNES."],
+      "question": "What was the first game Yoshi appeared in?",
+      "answer": "Super Mario World"
+    }  
+  ],
+  "general":[
+    {
+      "id": 100,
+      "question": "Name \"the land of a thousand lakes\"",
+      "answer": "Finland",
+      "info": "Finland is a country in Northern Europe. Sweden borders it to the northwest, Estonia to the south, Russia to the east, and Norway to the north. Finland is part of the European Union with its capital city being Helsinki. With a population of 5.5 million people, it has over 187,000 lakes. The thousands of lakes in Finland are the reason why the country's nickname is \"the land of a thousand lakes.\""
+    },
+    {
+      "id": 101,
+      "question": "Who was the winner of FIFA 2018?",
+      "answer": "France",
+      "info": "France 4 - 2 Croatia"
+    },
+    {
+      "id": 102,
+      "question": "What is the largest ocean in the world?",
+      "answer": "Pacific",
+      "info": "The Pacific Ocean is the largest and deepest of the world ocean basins. Covering approximately 63 million square miles and containing more than half of the free water on Earth, the Pacific is by far the largest of the world's ocean basins."
+    },
+    {
+      "id": 103,
+      "question": "Who gifted the Statue Of Liberty?",
+      "answer": "France",
+      "info": "The Statue of Liberty was a gift from the French people commemorating the alliance of France and the United States during the American Revolution. Yet, it represented much more to those individuals who proposed the gift."
+    },
+    {
+      "id": 104,
+      "question": "Which country is known as the \"Land Of The Rising Sun\"?",
+      "answer": "Japan",
+      "info": "The title stems from the Japanese names for Japan, Nippon/Nihon, both literally translating to \"the suns origin\"."
+    },
+    {
+      "id": 105,
+      "question": "What's known as the \"Playground of Europe\"?",
+      "answer": "Switzerland",
+      "info": "It comes from the title of a book written in 1870 by Leslie Stephen (father of Virginia Woolf) detailing his exploits of mountain climbing (not skiing) of which sport he was one of the pioneers and trekking or walking."
+    },
+    {
+      "id": 106,
+      "question": "Which country is known as the \"Land of Thunderbolt\"?",
+      "info": "\"Bhutan is known as the \\\"Land of Thunder Dragon\\\" or \\\"Land of Thunderbolt\\\" due to the violent and large thunderstorms that whip down through the valleys from the Himalayas. The dragon reference was due to people thinking the sparkling light of thunderbolts was the red fire of a dragon.\""
```

```suggestion
      "info": "Bhutan is known as the \"Land of Thunder Dragon\" or \"Land of Thunderbolt\" due to the violent and large thunderstorms that whip down through the valleys from the Himalayas. The dragon reference was due to people thinking the sparkling light of thunderbolts was the red fire of a dragon."
```

Appears to be an autoformatted mistake after pasting into pycharm.